openfl/federated/task/fl_model.py

```diff
@@ -75,10 +76,17 @@ def __init__(self, build_model, optimizer=None, loss_fn=None, **kwargs):
             self.runner.validate = lambda *args, **kwargs: build_model.validate(
                 self.runner, *args, **kwargs
             )
+        # TODO: not used anywhere - should it be removed?
         if hasattr(self.model, "train_epoch"):
             self.runner.train_epoch = lambda *args, **kwargs: build_model.train_epoch(
                 self.runner, *args, **kwargs
             )
+
+        # Used to hook the training function when debugging locally
```

Comment:
Is this change related to the FedCurv task runner example?

Reply:
I needed it to debug fedcurv, I don't mind omitting it if you don't think it should be included in this PR

Reply:
Ohh, I think it's okay to keep then. In fact, probably an overdue update since most of new runners (definitely the case for torch) use `train_` instead of `train_epoch`